claude-managed-agents-guide.html

```
@@ -0,0 +1,2171 @@
+              </div>
+              <div class="step-content">
+                <div class="step-title">CLI と SDK をインストールする</div>
+                <p>
+                  Anthropic は <code>ant</code> という CLI ツールを提供しています。SDK
+                  と合わせてインストールします。
+                </p>
+
+                <div class="code-block">
+                  <div class="code-header">
+                    <span class="code-lang">bash — CLI インストール</span>
+                    <button class="code-copy" onclick="copyCode(this)">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)